### DISCUSSION TITLE
How to add Legal Pages with Heroku deployment?

### DISCUSSION THREAD

**Discussion by masteradit** [getfider/fider#955] [Q&A] [answered]
According to the [docs](https://getfider.com/docs/how-to-show-legal-pages), it can only be done when the deployment is done using Docker. I am using Heroku to deploy Fider. Is there any way to add Legal Pages with this method?

**Comment by goenning** [comment, marked as answer]
I think if you modify terms.md and privacy.md on your fork and push it'll work.
